.autover/changes/b188d010-090a-4ead-b9cf-8f82d7c236ed.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.AppSyncEvents",
+      "Type": "Patch",
```

Comment:
The increment type should have been `Major` instead of `Patch` since this is a new package. You should have used `--increment-type Major` parameter while using `autover` (It's not well documented though, 😞)

Reply:
Also, I was asking to validate if validate if `AppSyncAuthorizerEvent` (and `AppSyncAuthorizerResult`) are out of scope for now, refer such event and result type at https://www.npmjs.com/package/@types/aws-lambda?activeTab=code.

Reply:
> The increment type should have been `Major` instead of `Patch` since this is a new package. You should have used `--increment-type Major` parameter while using `autover` (It's not well documented though, 😞)

@philasmar Please review comment https://github.com/aws/aws-lambda-dotnet/pull/1939#discussion_r1985798170. Should we be specifying version in `.csproj `file for 1st version? Also should the `--increment-type`  be `Major` for this use case.